packages/web/src/components/track/CardTitle.tsx

```diff
@@ -0,0 +1,81 @@
+import { FeatureFlags, PremiumConditions } from '@audius/common'
+import { IconCollectible, IconSpecialAccess } from '@audius/stems'
+import cn from 'classnames'
+
+import Tooltip from 'components/tooltip/Tooltip'
+import { useFlag } from 'hooks/useRemoteConfig'
+import HiddenTrackHeader from 'pages/track-page/components/HiddenTrackHeader'
+
+import styles from './GiantTrackTile.module.css'
```

Comment:
Nit: can we make each component manage its own styles?

Reply:
was lazy lol. will keep track of this and hopefully get to it